src/core/Song.cpp

```diff
@@ -339,14 +338,17 @@ void Song::processNextBuffer()
 				// if looping-mode is enabled and we have got
 				// past the looping range, return to the 
 				// beginning of the range
+				if( m_playPos[m_playMode] == tl->loopEnd() - 1 )
+				{
+					emit updateSampleTracks();
+				}
 				if( m_playPos[m_playMode] >= tl->loopEnd() )
 				{
 					m_playPos[m_playMode].setTicks( tl->loopBegin().getTicks() );
 
 					m_elapsedMilliSeconds = 
 						( ( tl->loopBegin().getTicks() ) * 60 * 1000 / 48 ) / 
```

Comment:
No. This will make it harder to deduce where 1250 came from in the first place. The compiler does operations on real numbers for us.